__tests__/app.test.js

```diff
@@ -129,6 +153,30 @@ describe('app testing', () => {
                 })
             })
         })
+        test('200: accepts a sort_by query', () => {
+            return request(app)
+            .get('/api/articles?sort_by=article_id')
+            .expect(200)
+            .then(({ body: {articles} }) => {
+                expect(articles).toBeSortedBy('article_id', {descending: true})
+            })
+        })
+        test('200: accepts an order query', () => {
+            return request(app)
+            .get('/api/articles?order=ASC')
+            .expect(200)
+            .then(({ body: {articles} }) => {
+                expect(articles).toBeSortedBy('created_at', {ascending: true})
+            })
+        })
+        test('200: accepts a topic query', () => {
+            return request(app)
+            .get('/api/articles?topic=cats')
+            .expect(200)
+            .then(({ body: {articles} }) => {
+                expect(articles[0].topic).toBe('cats')
+            })
```

Comment:
This should test that all articles have the topic of cats instead of just the first one